ABloom/ABloom/Presentation/CoreViews/QnA/View/QuestionMainView.swift

```diff
@@ -56,15 +33,74 @@ struct QuestionMainView: View {
             AnswerCheckView(num: content)
           }
         })
-        
-        .frame(maxWidth: .infinity, maxHeight: .infinity)
         .background(backWall())
       }
       .background(backgroundDefault())
     }
   }
 }
 
+extension QuestionMainView {
+  private var headerView: some View {
+    // 헤더
+    HStack {
+      Text("우리의 문답")
+        .fontWithTracking(.title3Bold)
+        .padding([.leading], 20)
+      Spacer()
+      // FIXME: 다른 뷰로 전환 시에는 다른 방식으로 처리해야함
+      NavigationLink(value: 0) {
+        Image("pencil_write_fill")
+      }
+      .padding([.trailing], 17)
+    }
+    .padding([.top], 20)
```

Comment:
사소하지만 여러가지 패딩 값을 한 번에 조정할 것이 아니라면 대괄호를 빼는 것은 어떨까요?
```suggestion
      .padding(.trailing, 17)
    }
    .padding(.top, 20)
```

Reply:
앗! 네네 확인하지 못한 부분이네요. 수정하겠습니다.
얼라인 맞춰서 오늘 저녁에 다시 올릴게요~